code/modules/projectiles/projectile.dm

```diff
@@ -658,7 +658,7 @@
 	bumped = TRUE
 	if(ismob(A))
 		var/mob/M = A
-		if(isliving(A))
+		if(isliving(A) && A.stat == DEAD) //Dead people are limp and unable to be moved to block bullets to stop endless monkey laser proof shields
```

Comment:
Doesn't this actually make it only work if A is dead?

Reply:
Should

Reply:
WAIT YE

Reply:
fixed